### PR TITLE
Homepage

### PR DESCRIPTION
**Pull request by stutisk**
added Homepage and product page
made API calls to get categories and product items 
These pages contain 
Navbar , Footer, Asidebar ,Card , components

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,12 +1,32 @@
...
+// import React from "react";
+// import Mockman from "mockman-js";
+// function MockAPI() {
+//   return (
+//     <div className="MockAPI">
+//       <Mockman />
+//     </div>
+//   );
+// }
+
```
Comment (Manpreet-hub) [inline src/App.js:31]: remove commented code if not required

File: src/Components/AsideBar/AsideBar.jsx [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+import React from "react";
+import "../../styles/ProductListing.css";
+const AsideBar = () => {
+    return (
+    <>
+      <div class="drawer-container product-drawer-container">
+                <div class="drawer ">
+                    <aside class="aside filter-aside">
+                        <div class="drawer-header ">
+                            <div class="filter-topic m-2t">
```
Comment (Manpreet-hub) [inline src/Components/AsideBar/AsideBar.jsx]: Here Class should be changed to className

File: src/Components/ProductListing/ProductListing.jsx [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+                alt={products.category}
+                className="image-product"
+              />
+            </div>
+            <div class="delivery details">
+              <div class="delivery-details m-5">
+              {products.brand}
+            
+              </div>
+            </div>
```
Comment (Manpreet-hub) [inline src/Components/ProductListing/ProductListing.jsx:41]: same in this file also (class should be changed to className)

**Review by Manpreet-hub — APPROVED** [review]
suggested some changes.

File: src/Components/Cards/Categories.jsx [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+
+      async  function getCategories  () {
+      try {
+        const res = await axios.get("/api/categories");
+        console.log(res.data)
+        setCategories(res.data.categories);
+      } catch (error) {
+        console.log(error);
+      }
+    }
```
Comment (Hardiegogo) [inline src/Components/Cards/Categories.jsx:18]: You can abstract these functions out in a different folder like Category-utils.

Comment (stutisk) [reply src/Components/Cards/Categories.jsx:18]: will surely try it this way ,thanx

**Pull request merged**